Description: Minor man page fixes
 This patch fixes some minor man page issues
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=527607

Index: dia-0.97.3+git20160930/doc/eu/dia-1.xml
===================================================================
--- dia-0.97.3+git20160930.orig/doc/eu/dia-1.xml
+++ dia-0.97.3+git20160930/doc/eu/dia-1.xml
@@ -186,7 +186,7 @@
       </varlistentry>
       <varlistentry>
 	<term><option>-t <replaceable>FORMAT</replaceable></option>
-	  <option>--export-to-format=<replaceable>FORMAT</replaceable></option></term>
+	  <option>--filter=<replaceable>FORMAT</replaceable></option></term>
 	<listitem>
 	  <para>Export loaded file in FORMAT and exit.  Format are
 	  described below.</para>
Index: dia-0.97.3+git20160930/doc/fr/dia-1.xml
===================================================================
--- dia-0.97.3+git20160930.orig/doc/fr/dia-1.xml
+++ dia-0.97.3+git20160930/doc/fr/dia-1.xml
@@ -190,7 +190,7 @@
       </varlistentry>
       <varlistentry>
 	    <term><option>-t <replaceable>FORMAT</replaceable></option>
-	    <option>--export-to-format=<replaceable>FORMAT</replaceable></option></term>
+	    <option>--filter=<replaceable>FORMAT</replaceable></option></term>
 	    <listitem>
             <para>Exporte le fichier chargé dans le FORMAT et quitte.  Les
             FORMATS D&apos;EXPORTATION sont décrits ci-dessous.</para>
